app/connect.py

```diff
@@ -0,0 +1,102 @@
+import os
+import psycopg2
+
+from app import schema
+from instance.config import Config
+
+
+
+def init_db():
+    """ setup database connection"""
+    url = os.getenv('DATABASE_URL')
+    conn = psycopg2.connect("dbname=questioner user=emdee password=arif@123")
+    conn.commit()
```

Comment:
If you are not using the url from the database environment you should remove it

Reply:
I have removed the unused variables